src/skirmishbunker/DetailedRoof.py

```diff
@@ -0,0 +1,186 @@
+import cadquery as cq
```

Comment:
Reminder to myself.
There is an example roof file that will have to be updated to instance DetailedRoof.

This file will also need a license block. Although I doubt I had it in place originally